src/Components/RegistrationProgress/FirstStep/SuccessPage.cy.js

```diff
@@ -0,0 +1,19 @@
+import { singleActivationKeyInterceptor } from '../../../../cypress/support/interceptors';
+import SuccessPage from './SuccessPage';
+
+const CHECKMARK_SVG =
+  'M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z';
+
+describe('Rendering tests.', () => {
+  beforeEach(() => {
+    singleActivationKeyInterceptor.keys();
+    cy.mountWithContext(SuccessPage);
+  });
+  it('Success page renders correctly', () => {
+    cy.get('body');
+    // close button displayed
+    cy.get('button').should('include.text', 'Close');
+    // correct icon displayed
+    cy.get('path').should('have.attr', 'd', CHECKMARK_SVG);
```

Comment:
We usually just assert that the container with the PF/EmptyStateIcon class name exists, but I like your more precise way to assert the correct SVG path. We could improve it a bit more with:

```suggestion
// eslint-disable-next-line rulesdir/disallow-fec-relative-imports
import { EMPTY_STATE_ICON } from '@redhat-cloud-services/frontend-components-utilities';
...
    cy.get(EMPTY_STATE_ICON)
      .find('path')
      .should('have.attr', 'd', CHECKMARK_SVG);```